qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -4,7 +4,7 @@
 from tasks.cephfs.cephfs_test_case import CephFSTestCase
 
 
-logger = getLogger(__name__)
+log = getLogger(__name__)
```

Comment:
Since this is not used in this file, could we just remove it ?

Reply:
The log variable is being used in one of the follow-up PRs.

Reply:
Please make this change in the PE where its being used.

Reply:
Okay. The change's been removed from this PR.

Reply:
@vshankar I forgot to tag you on last comment.